2021-05-04 15:48:01,388 INFO    MainThread:74 [wandb_setup.py:_flush():69] setting env: {}
2021-05-04 15:48:01,389 INFO    MainThread:74 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-04 15:48:01,389 INFO    MainThread:74 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8d3153ad3bb912131270614820cf8ed59c9d94d4'}
2021-05-04 15:48:01,390 INFO    MainThread:74 [wandb_init.py:_log_setup():336] Logging user logs to /content/drive/My Drive/Trash/code/wandb/run-20210504_154801-18naje14/logs/debug.log
2021-05-04 15:48:01,390 INFO    MainThread:74 [wandb_init.py:_log_setup():337] Logging internal logs to /content/drive/My Drive/Trash/code/wandb/run-20210504_154801-18naje14/logs/debug-internal.log
2021-05-04 15:48:01,391 INFO    MainThread:74 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fb10fd52310>
2021-05-04 15:48:01,391 INFO    MainThread:74 [wandb_init.py:init():369] calling init triggers
2021-05-04 15:48:01,391 INFO    MainThread:74 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-04 15:48:01,391 INFO    MainThread:74 [wandb_init.py:init():418] starting backend
2021-05-04 15:48:01,392 INFO    MainThread:74 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-04 15:48:01,431 INFO    MainThread:74 [backend.py:ensure_launched():123] starting backend process...
2021-05-04 15:48:01,469 INFO    MainThread:74 [backend.py:ensure_launched():128] started backend process with pid: 409
2021-05-04 15:48:01,471 INFO    MainThread:74 [wandb_init.py:init():423] backend started and connected
2021-05-04 15:48:01,509 INFO    MainThread:74 [wandb_init.py:init():465] updated telemetry
2021-05-04 15:48:01,517 INFO    MainThread:74 [wandb_init.py:init():484] communicating current version
2021-05-04 15:48:02,363 INFO    MainThread:74 [wandb_init.py:init():489] got version response 
2021-05-04 15:48:02,363 INFO    MainThread:74 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-04 15:48:02,531 INFO    MainThread:74 [wandb_init.py:init():522] starting run threads in backend
2021-05-04 15:48:04,404 INFO    MainThread:74 [wandb_run.py:_console_start():1554] atexit reg
2021-05-04 15:48:04,412 INFO    MainThread:74 [wandb_run.py:_redirect():1401] redirect: SettingsConsole.WRAP
2021-05-04 15:48:04,413 INFO    MainThread:74 [wandb_run.py:_redirect():1444] Wrapping output streams.
2021-05-04 15:48:04,413 INFO    MainThread:74 [wandb_run.py:_redirect():1468] Redirects installed.
2021-05-04 15:48:04,414 INFO    MainThread:74 [wandb_init.py:init():546] run started, returning control to user process
2021-05-04 15:48:04,414 INFO    MainThread:74 [wandb_config.py:__setitem__():141] config set learning_rate = 0.01 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fb10a812a50>>
2021-05-04 15:48:04,415 INFO    MainThread:74 [wandb_run.py:_config_callback():779] config_cb learning_rate 0.01 None
2021-05-04 15:48:04,415 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:48:04,416 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:48:04,423 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:48:10,999 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:48:10,999 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:48:12,361 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:48:34,480 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:48:34,481 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:48:34,488 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:48:34,520 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:48:34,520 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:48:34,527 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:48:34,553 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:48:34,553 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:48:45,410 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:48:45,415 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:48:45,416 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:48:53,102 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:48:53,131 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:48:53,132 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:49:04,632 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:49:08,144 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:49:08,144 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:49:16,611 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:49:16,626 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:49:16,627 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:49:17,235 INFO    MainThread:74 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 15:49:26,045 INFO    MainThread:74 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 15:49:26,045 INFO    MainThread:74 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 15:49:40,809 INFO    MainThread:74 [wandb_run.py:_atexit_cleanup():1524] got exitcode: 0
2021-05-04 15:49:40,811 INFO    MainThread:74 [wandb_run.py:_restore():1496] restore
2021-05-04 15:49:41,007 INFO    MainThread:74 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 650
}

2021-05-04 15:49:41,876 INFO    MainThread:74 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 650
  total_bytes: 8000
}

2021-05-04 15:49:42,159 INFO    MainThread:74 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8000
  total_bytes: 8000
}

2021-05-04 15:49:42,264 INFO    MainThread:74 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8000
  total_bytes: 8000
}

2021-05-04 15:49:42,367 INFO    MainThread:74 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8000
  total_bytes: 8000
}

2021-05-04 15:49:42,470 INFO    MainThread:74 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8000
  total_bytes: 8000
}

2021-05-04 15:49:42,573 INFO    MainThread:74 [wandb_run.py:_wait_for_finish():1646] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8000
  total_bytes: 8000
}

2021-05-04 15:49:43,862 INFO    MainThread:74 [wandb_run.py:_show_files():1868] logging synced files
